tests/configs/gpii.tests.acceptance.localInstall.config.json

```diff
@@ -9,5 +9,5 @@
             }
         }
     },
-    "mergeConfigs": "%universal/gpii/configs/gpii.config.development.all.local.json"
+    "mergeConfigs": "%universal/gpii/configs/gpii.config.all.development.dr.production.json"
```

Comment:
It seems that this file is now identical to "gpii.tests.acceptance.dynamicDeviceReporter.config.json" - so it seems sensible to remove one of them.. perhaps this one and then use the dynamicDeviceReporter instead and update references and docs accordingly